relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -376,9 +389,43 @@ pub(crate) fn extract_tags(span: &Span, config: &Config) -> BTreeMap<SpanTagKey,
         span_tags.insert(SpanTagKey::StatusCode, status_code);
     }
 
+    if let Some(end_time) = span.timestamp.value() {
+        if let Some(initial_display) = initial_display {
+            if end_time <= &initial_display {
+                span_tags.insert(SpanTagKey::TimeToInitialDisplay, "ttid".to_owned());
+            }
+        }
+        if let Some(full_display) = full_display {
+            if end_time <= &full_display {
+                span_tags.insert(SpanTagKey::TimeToFullDisplay, "ttfd".to_owned());
+            }
+        }
+    }
+
     span_tags
 }
 
+/// Finds time-to-initial/full-display spans and returns its end times.
+fn display_times(spans: &[Annotated<Span>]) -> (Option<Timestamp>, Option<Timestamp>) {
```

Comment:
Maybe instead of the tuple here it would be clearer to create a struct and set the fields with proper names in it. 
It can be easy to mistaken the order of the returned values just looking at the function signature.

Reply:
Good point. I refactored this to use `.find()` instead of writing a homemade loop now, let me know what you think.

Reply:
yeah, nice!